saga-core/src/main/java/io/servicecomb/saga/core/ForwardRecovery.java

```diff
@@ -41,7 +41,14 @@ public void apply(SagaTask task, SagaRequest request) {
             request.serviceName(),
             e
         );
+        try {
+          Thread.sleep((int) (request.failRetryDelaySeconds() * 1000));
+        } catch (InterruptedException ie) {
+          break;
```

Comment:
why not catch interruption outside while loop, in order to avoid break again?

Reply:
Done